include/deal.II/hp/refinement.h

```diff
@@ -103,9 +103,8 @@ namespace hp
    * Vector<float> estimated_error_per_cell (triangulation.n_active_cells());
    * KellyErrorEstimator<dim>::estimate(hp_dof_handler,
    *                                    QGauss<dim-1> (quadrature_points),
-   *                                    typename FunctionMap<dim>::type(),
-   *                                    solution,
-   *                                    estimated_error_per_cell);
+   *                                    std::map<types::boundary_id, const
+   * Function<dim, Number> *>(), solution, estimated_error_per_cell);
```

Comment:
:-(

Reply:
You can't break this manually such that no line extends 80 characters?
```
   * KellyErrorEstimator<dim>::estimate(
   *   hp_dof_handler,
   *   QGauss<dim-1> (quadrature_points),
   *   std::map<types::boundary_id, const Functions<dim, Number> *>(),
   *   solution,
   *   estimated_error_per_cell);
```
works just fine for me.

Reply:
good point, thanks.